src/shared/components/Accordion/Item.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { PropsWithChildren } from "react";
+import { PropsWithChildren } from "react";
```

Comment:
@2div import forwardRef from "react" 
```suggestion
import { forwardRef, PropsWithChildren } from "react";
```
and delete React on line 9
```tsx
export const Item = forwardRef<HTMLDivElement, ItemProps>(function Item(
```

Reply:
Hi @GridisBlueFeathers  Thank you for you response.

I did the suggested change and still the error not stop.

![image](https://github.com/dest-nft/dest.sh/assets/96177680/d48957bf-678a-46d7-a6ae-b171f316e9b9)

The problem in NoiseBackground.tsx  and says :

Cannot find module '../../../../public/images/noise.png' or its corresponding type declarations.

however the image is imported. i can not find out why it still giving error.

![image](https://github.com/dest-nft/dest.sh/assets/96177680/3f764d42-8917-48c5-92f6-f4f9171f1e29)

Reply:
@2div Could you please try to commit suggested changes and let's check CI 😸

Reply:
@yura2100 I committed suggested changes , Please check it :)